docs/api/options.mdx

```diff
@@ -0,0 +1,118 @@
+---
+sidebar_position: 1
+---
+
+import { BasicDemo } from '../../website/src/components/demos';
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+# Options
+
+## Autoplay
+
+The carousel can advance on its own with a specified interval measured in milliseconds. An `autoplayInterval` without the `autoplay` prop being true will not do anything.
+
+### Details
+
+| Prop Name          | Type    | Default Value |
+| :----------------- | :------ | :------------ |
+| `autoplay`         | boolean | `false`       |
+| `autoplayInterval` | number  | `3000`        |
+
+### Example
+
+<BasicDemo className="standard-demo" autoplay={true} />
+
+---
+
+## Scroll
+
+How far the carousel should move when its goForward() and goBack() methods are called.
+
+### Details
+
+| Prop Name        | Type                          | Default Value |
+| :--------------- | :---------------------------- | :------------ |
+| `scrollDistance` | "slide" \| number \| "screen" | `slide`       |
+
+Using a number will cause the carousel to scroll that many pixels each time.
+Using "slide" will advance the carousel one slide each time.
+Using "screen" will advance the carousel by the width of the visible carousel.
```

Comment:
I think this might be the same issue as the indicator issue; I'm guessing you changed the window size to simulate mobile and the component didn't recalculate its number of slides relative to the screen size. This should be solved with the resize listener in 3f4df56

Reply:
There was a different bug similar to this on mobile devices that should be fixed in c2e59d2. I rounded the manual scrolling check since something was pushing the scroll indices to be some decimal point off, which would cause some slides to skip when pressing next or not move when pressing back.